package.json

```diff
@@ -88,7 +94,7 @@
     "browserify": "*",
     "browserify-versionify": "^1.0.6",
     "coveralls": "^2.13.1",
-    "grunt": "~0.4.2",
+    "grunt": "^0.4.2",
```

Comment:
guessing it rolled up to 0.7 or something crazy. this is definitely the conservative option..